tetra/app.py

```diff
@@ -45,25 +45,19 @@ def on_get(self, req, resp):
         resp.body = json.dumps(version)
 
 
-class TetraAPI(falcon.API):
-
-    RESOURCES = [
-        BuildResource(),
-        ResultsResource(),
-        ResultResource(),
-        LastCountByStatusResultsResource(),
-        LastCountByTestNameResultsResource(),
-        ProjectResultsResource(),
-        BuildsResource(),
-        ProjectsResource(),
-        ProjectResource(),
-        VersionResource(),
-    ]
-
-    def __init__(self):
-        super(TetraAPI, self).__init__()
-        for resource in self.RESOURCES:
-            self.add_route(resource.ROUTE, resource)
-
-
-application = TetraAPI()
+application = falcon.App()
```

Comment:
Q:
Just curious, why go from inheritance to this composition like setup?

Reply:
that's a change in the falcon API, or at least the default way the docs say it now (but I also _think_ the inheritance based setup was removed, unfortunately I can't find the page I came across this though).